scripts/generateESLintTest.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
Mmk, can't say I'm the biggest fan, but if it makes it easier for your team then go for it 👍